app/components/pages/SubmissionWizard/steps/Files/index.test.js

```diff
@@ -0,0 +1,107 @@
+import { onFileDrop } from '.'
+
+const fakeFunction = () => {}
+const fakeResponse = fileName => ({
+  data: {
+    uploadManuscript: { meta: { tile: '' }, files: [{ filename: fileName }] },
+  },
+})
+
+describe('FilesPageContainer', () => {
+  it('onFileDrop sets new manuscript file value when there is no validation error and no existing manuscript file', async () => {
+    const values = {
+      files: [],
+    }
+    const setValues = (fieldName, value) => (values[fieldName] = value)
+    const files = [
+      {
+        name: 'document1.docx',
+      },
+    ]
+    await onFileDrop(
```

Comment:
It's hard to see what's going on here. Can we look at using an object/multiple objects instead for the parameters of `onFileDrop`?